### PR TITLE
Update linkfree-cli.mdx

### LINKED ISSUE #8845: [DOCS] pages/docs/environments/linkfree-cli.mdx should be changed to BioDrop as well
### Description

linkfree-cli still follows the name which should now follow the name of BioDrop
the path to the file is pages/docs/environments/linkfree-cli.mdx

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by sanyamx**
Changed it to match current name "BioDrop"

## Fixes Issue
closes #8845 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I would like to know more about if there are any future consideration of changing the CLI name as well so as to create a uniformity in the documents. If yes i would like to update the document more.
Thanks! Please review and let me know if there are any changes required!
Happy Reviewing

### CODE REVIEW

File: pages/docs/environments/linkfree-cli.mdx [hunk 1]
```diff
@@ -31,9 +31,9 @@ npm run cli
 
 ![LinkFree CLI main prompt](https://user-images.githubusercontent.com/51878265/220891756-10861090-e719-4687-b4fb-f7bef2826934.png)
```
Comment (sital002) [inline pages/docs/environments/linkfree-cli.mdx:32]: Please change this as well.

**Review by sital002 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Check this comment: https://github.com/EddieHubCommunity/BioDrop/issues/8845#issuecomment-1705529907

**Comment by sanyamx** [comment]
> Hey,
> 
> Check this comment before you proceed further: [#8845 (comment)](https://github.com/EddieHubCommunity/BioDrop/issues/8845#issuecomment-1705529907)

Sure! i will just update the name where there is LinkFree to BioDrop, you can do the rest!
I will be more than happy to help if i can contribute to CLI as well, please let me know if i can contribute!

**Comment by Pradumnasaraf** [comment]
Hey,

Please read the comment https://github.com/EddieHubCommunity/BioDrop/issues/8845#issuecomment-1705529907

> 1st the CLI will get a rename, then the docs will proceed (Hopefully soon). Also, I will be self-assigning this issue to make this change.

Thanks. I am still in the process of renaming. You can look at the project and see the way you can contribute.

**Pull request closed without merging**